vpc/service/service.go

```diff
@@ -248,6 +253,12 @@ func Run(ctx context.Context, config *Config) error {
 		return errors.New("The worker role must be non-empty")
 	}
 
+	validCNRegex, err := regexp.Compile(config.ValidCNs)
+	if err != nil {
+		panic(fmt.Errorf("Error parsing ValidCN Regexes: %w", err))
```

Comment:
Any reason not to just return this error from here?

Reply:
Nope